src/sections/dataset/dataset-citation/DatasetCitation.tsx

```diff
@@ -0,0 +1,69 @@
+import { Col, Icon, Row, Tooltip } from 'dataverse-design-system'
+import styles from './DatasetCitation.module.scss'
+import { useTranslation } from 'react-i18next'
+import { Citation, DatasetStatus } from '../../../dataset/domain/models/Dataset'
+
+interface DatasetCitationProps {
+  citation: Citation
+  status: DatasetStatus
+  version: string | null
+}
+interface CitationDatasetStatusProps {
+  status: DatasetStatus
+}
+
+function CitationDescription({ citation, status, version }: DatasetCitationProps) {
+  return (
+    <span>
+      {citation.citationText}, {citation.pidUrl}, {citation.publisher}
+      {version && ', V' + version.substring(0, 1)}
+      {citation.unf && ', ' + citation.unf}
+      {status !== DatasetStatus.PUBLISHED && <CitationDatasetStatus status={status} />}
+    </span>
+  )
+}
+function CitationDatasetStatus({ status }: CitationDatasetStatusProps) {
+  const { t } = useTranslation('dataset')
+  return (
+    <span>
+      {', '}
+      {t(status)} <Tooltip placement={'top'} message={t(`${status}.description`)} />
+    </span>
+  )
+}
+
+export function DatasetCitation({ citation, status, version }: DatasetCitationProps) {
```

Comment:
Some tweaks on the citation display taking the JSF as reference

1. It should be aligned at the left to the rest of the elements
![Captura de pantalla 2023-05-22 a las 12 43 57](https://github.com/IQSS/dataverse-frontend/assets/23359572/eeba2453-6f08-41b0-b631-714904a0f880)

2. We need to use the full space at the right for the text
![Captura de pantalla 2023-05-22 a las 12 48 51](https://github.com/IQSS/dataverse-frontend/assets/23359572/a1233592-b7f8-4227-b75f-20b3265d563d)
3. We need to add padding at the bottom as well, to center the content and to leave some space at the botton between the icon and the border of the box
![image](https://github.com/IQSS/dataverse-frontend/assets/23359572/8a52722c-661e-4bb3-94e6-1cbaefb558a0)
4. We need to add some space between the citation and the summary

Reply:
I fixed the styles for 1 and 3 above, but for 2, padding on the right side of the citation, I couldn't find a place where the padding in the new component was different than the JSF.  Is it possible that it's just a wrapping issue? In that example the url is long, so goes to the next line.  Here is an example of what it looks like with a long title: 
Is there something in the JSF styles that I'm missing?

<img width="864" alt="Screenshot 2023-05-22 at 3 38 33 PM" src="https://github.com/IQSS/dataverse-frontend/assets/675224/059896fc-d8cd-435c-965a-9d89ba8fe301">

Reply:
Ok, the 2 seems to be because the url is long 👍 

I added a commit to add the padding at the bottom, from the point 3, because it was the only thing missing. I'm going to approve this. Thanks!